transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/SchedulingConfigModule.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.runtime.scheduling;
+
+import com.google.android.datatransport.runtime.scheduling.jobscheduling.SchedulerConfig;
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public abstract class SchedulingConfigModule {
+
+  @Provides
+  static SchedulerConfig config() {
+    return SchedulerConfig.builder()
+        .setDelta(30000)
+        .setMaxAllowedTime(100000000)
```

Comment:
can you extract delta and maxAllowed to constants that have meaningful names? e.g `THIRTY_SECONDS` and whatever that 1 hundred million is(or maybe change it to `TWENTY_FOUR_HOURS`